dnsapi/dns_dnsservices.sh

```diff
@@ -204,7 +204,8 @@ deleteRecord() {
   fulldomain=$1
   txtvalue=$2
 
-  if [ "$(echo "$fulldomain" | grep "_acme-challenge")" = "" ]; then
+  # Fix for acmetest to limit acme.sh to only work on _acme-challenge and acmeTestXYzRandomName in GitHub actions
+  if [ "$(echo "$fulldomain" | grep "_acme-challenge\|acmetestXyzRandomName.github-test")" = "" ]; then
```

Comment:
don't do that.

It's required that you can add and remove txt record for any subdomain.domain.com.  Not only for `_acme-challenge`

Reply:
It was a simple check to avoid removing non ACME related valid records.

Removing and have pushed change.

Reply:
Otherwise it seems to work now with acmetest - Solaris is running from above push, but letting this push run its course.

Thanks for fixing acmetest - very fustrating to deal with random errros at random times in this end :-)